plugin.json

```
@@ -30,6 +30,27 @@
         "type": "text",
         "default": "/pages/dashboard",
         "help_text": "Relative Path to the Riff Stats page."
+      },
+      {
+        "key": "ReminderText",
+        "display_name": "Reminder Text:",
+        "type": "text",
+        "default": "A gentle reminder to fill the survey.",
+        "help_text": "Enter the message to be sent to users who have not responded to a survey yet."
```

Let's update this text to better represent how this message will be used.

```suggestion
        "help_text": "Enter the message to be sent to users as a reminder for filling pending surveys."
```

---

Done.